Packs/CyberArkPAS/Integrations/CyberArkPAS/CyberArkPAS.py

```diff
@@ -0,0 +1,285 @@
+    raws = []
+    cyberark_ec = []
+    raw_response = client.add_account(user_name=args.get('user_name'), address=args.get('address'),
+                                      platform_id=args.get('platform_Id'), safe_name=args.get('safe_name'),
+                                      name=args.get('name'), secret=args.get('secret'),
+                                      secret_type=args.get('secret_type'),
+                                      platform_account_properties=args.get('platform_account_properties'),
+                                      automatic_management_enabled=args.get('automatic_management_enabled'),
+                                      manual_management_reason=args.get('manual_management_reason'),
+                                      remote_machines=args.get('remote_machines'),
+                                      access_restricted_to_remote_machines=args.get(
+                                          'access_restricted_to_remote_machines'))
+    if raw_response:
+        raws.append(raw_response)
+        cyberark_ec.append({
+            'AccountName': raw_response.get('name'),
+            'UserName': raw_response.get('userName'),
+            'PlatformID': raw_response.get('platformId'),
+            'SafeName': raw_response.get('safeName'),
+            'AccountID': raw_response.get('id'),
+            'CreatedTime': raw_response.get('createdTime')
+        })
+
+    if not raws:
+        return_outputs(f'{INTEGRATION_NAME} - Could not create the new Account', {}, {})
+
+    return CommandResults(
+        outputs_prefix='CyberArk.Accounts',
+        outputs_key_field='AccountID',
+        outputs=cyberark_ec,
+        raw_response=raw_response,
+        readable_output=tableToMarkdown("CyberArk PAS New Account", cyberark_ec)
+    )
+
+
+def delete_account(client, args):
+    raws = []
+    cyberark_ec = []
+    raw_response = client.delete_account(account_id=args.get('account_id'))
+
+    if raw_response:
+        raws.append(raw_response)
+        cyberark_ec.append({
+            raw_response
+        })
+
+    return "Account is Deleted"
+
+
+def main():
+    """
+        PARSE AND VALIDATE INTEGRATION PARAMS
+    """
+    params = demisto.params()
+    username = params.get('credentials', {}).get('identifier', '')
+    password = params.get('credentials', {}).get('password', '')
+    base_url = params['url'][:-1] if (params['url'] and params['url'].endswith('/')) else params['url']
+    verify_certificate = not params.get('insecure', False)
+    proxy = params.get('proxy', False)
+
+    demisto.debug(f'Command being called is {demisto.command()}')
+
+    try:
+        client = Client(
+            base_url=base_url,
+            verify=verify_certificate,
+            proxy=proxy,
+            ok_codes=(200, 201, 204),
+            headers={'accept': "application/json"}
+        )
+
+        if not (demisto.getIntegrationContext().get('valid_until') or demisto.getIntegrationContext().get('valid_until')):
```

Comment:
```suggestion
        if not demisto.getIntegrationContext().get('valid_until') or int(time.time()) > demisto.getIntegrationContext().get('valid_until'):
```

We can combine this if statement and the one in 242-243 into a single statement.
Correct me if I'm wrong but we basically want to `login` in case a `valid_untill` does not exist or that the current time higher than the one in `valid_untill`.

Sub question - could you please provide an example of the `valid_until` value? we need to make sure it is in seconds and not milliseconds so the logic will be true.